README.md

```diff
@@ -37,50 +31,61 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 
 
 <!-- This heading should always match the name of the root level module (aka the repo name) -->
-## terraform-ibm-module-template
+## terraform-ibm-trusted-profile
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
+```hcl
+module "trusted_profile {
+  source                      = "terraform-ibm-modules/trusted-profile/ibm"
+  version                     = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
+  trusted_profile_name        = "example-profile"
+  trusted_profile_description = "Example Trusted Profile"
+
+  trusted_profile_policies = [{
+    roles = ["Reader", "Viewer"]
+    resources = [{
+      service           = "kms"
+    }]
+  }]
+
+  trusted_profile_claim_rules = [{
+    conditions = [{
+      claim    = "Group"
+      operator = "CONTAINS"
+      value    = "\"Admin\""
```

Comment:
Does this value really have to include quotes in it? Can we not require that?

Reply:
Yes, I tested without the escaped quotes and the provider errors out if it isn't formatted with them in.